.devcontainer/scripts/start.sh

```diff
@@ -3,5 +3,4 @@
 # Performs setup tasks _after_ the .devcontainer Docker container is created.
 
 # Install Node dependencies.
-yarn install
-yarn build
```

Comment:
We should leave this here for now, it only impacts the Docker container here FYI, but is consistent with the README instructions pertaining to Docker